tests/pages/logins-overview-page.test.tsx

```diff
@@ -27,22 +29,18 @@ describe("LoginsOverviewPage", () => {
     it("should display primary email identity first and all of them when clicking on the show more button", async () => {
       expect.assertions(7);
 
-      const sortedIdentities: SortedIdentities = {
-        emailIdentities: [
-          mockIdentities.primaryEmailIdentity,
-          mockIdentities.nonPrimaryEmailIdentity,
-        ],
-        phoneIdentities: [],
-        socialIdentities: [],
-      };
+      const identities = [
+        mockIdentities.primaryEmailIdentity,
+        mockIdentities.nonPrimaryEmailIdentity,
+      ];
+
+      const sortedIdentities: SortedIdentities = sortIdentities(identities);
```

Comment:
This kind of things has been changed across the tests file because now, the `GET /api/identities` doesn't send objects with `emailIdentities`, `primaryIdentities` or thinks like that, it only respond with `Identity[]`.